CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ All versions prior to 1.0.0 are untracked.
 
 ## [Unreleased]
 
+## [1.0.1] - 2024-04-18
+
+### Added
+- Added support for pre v1.0 signatures used in production. This is only provided for verification and replicates the experimental behavior at v0.2, bug for bug.
+- Added support for displaying fingerprints of certificates when using signing certificates
+
+### Fixed
+- Fix bug in CLI scripts where even if signature verification failed, the script would also output that verification passed.
```

Comment:
was the bug outputting that verification passed? or that it had a zero exit code?

Reply:
Both, let me edit to make it correct.